zebrad/tests/acceptance.rs

```diff
@@ -250,6 +272,70 @@ where
 
         Ok(self)
     }
+
+    fn spawn_lightwalletd_child(self, args: &[&str]) -> Result<TestChild<Self>> {
+        let dir = self.as_ref().to_owned();
+        let default_config_path = dir.join("lightwalletd-zcash.conf");
+
+        assert!(
+            default_config_path.exists(),
+            "lightwalletd requires a config"
+        );
+
+        // By default, launch a working test instance with logging,
+        // and avoid port conflicts.
+        let mut extra_args: Vec<_> = vec![
+            // the fake zcashd conf we just wrote
+            "--zcash-conf-path",
+            default_config_path
+                .as_path()
+                .to_str()
+                .expect("Path is valid Unicode"),
+            // the lightwalletd cache directory
+            //
+            // TODO: create a sub-directory for lightwalletd
+            "--data-dir",
+            dir.to_str().expect("Path is valid Unicode"),
+            // log to standard output
+            //
+            // TODO: if lightwalletd needs to run on Windows,
+            //       work out how to log to the terminal on all platforms
+            "--log-file",
+            "/dev/stdout",
+            // let the OS choose a random available wallet client port
+            "--grpc-bind-addr",
+            "127.0.0.1:0",
+            "--http-bind-addr",
+            "127.0.0.1:0",
+            // don't require a TLS certificate for the HTTP server
+            "--no-tls-very-insecure",
+        ];
+        extra_args.extend_from_slice(args);
+
+        self.spawn_child_with_command("lightwalletd", &extra_args)
+    }
+
+    fn with_lightwalletd_config(self, zebra_rpc_listener: SocketAddr) -> Result<Self> {
+        use std::fs;
+        use std::io::Write;
+
+        let lightwalletd_config = format!(
+            "\
+            rpcbind={}\n\
+            rpcport={}\n\
+            ",
+            zebra_rpc_listener.ip(),
+            zebra_rpc_listener.port(),
+        );
+
+        let dir = self.as_ref();
+        fs::create_dir_all(dir)?;
+
+        let config_file = dir.join("lightwalletd-zcash.conf");
+        fs::File::create(config_file)?.write_all(lightwalletd_config.as_bytes())?;
```

Comment:
Suggestion: I think this could be simplified to use [`fs::write`](https://doc.rust-lang.org/std/fs/fn.write.html)

```suggestion
        fs::write(config_file, lightwalletd_config.as_bytes())?;
```